python/native_functions.py

```diff
@@ -107,6 +107,17 @@ def special_print(val):
         print(display)
     return val
 
+def append_to_list(val, l):
+    l.append(val)
+    return l[:]
```

Comment:
Hmm, doesn't this modify the original list?

Reply:
Yes, that is intended.

Reply:
Seems like it would cause problems because the specifics of how values like lists are passed around are not specified